src/unary.js

```diff
@@ -8,6 +8,8 @@ import {Tensor} from './lib/tensor.js';
  * @param {Function} unaryFunc
  * @return {Tensor}
  */
+
+
```

Comment:
[nit] Is it normal to insert 2 blank lines or 1?

Reply:
ok, I have deleted these two blank lines